helios-services/src/main/java/com/spotify/helios/agent/SyslogRedirectingContainerDecorator.java

```diff
@@ -42,18 +44,34 @@ public SyslogRedirectingContainerDecorator(final String syslogHostPort) {
   }
 
   @Override
-  public void decorateHostConfig(HostConfig.Builder hostConfig) {
-    final List<String> binds = Lists.newArrayList();
-    if (hostConfig.binds() != null) {
-      binds.addAll(hostConfig.binds());
+  public void decorateHostConfig(Job job, Optional<String> dockerVersion,
+                                 HostConfig.Builder hostConfig) {
+    if (useSyslogRedirector(dockerVersion)) {
+      final List<String> binds = Lists.newArrayList();
+      if (hostConfig.binds() != null) {
+        binds.addAll(hostConfig.binds());
+      }
+      binds.add("/usr/lib/helios:/helios:ro");
+      hostConfig.binds(binds);
+    } else {
+      final ImmutableMap.Builder<String, String> logOpts = ImmutableMap.builder();
+
+      logOpts.put("syslog-address", "udp://" + syslogHostPort);
+      logOpts.put("syslog-facility", "local0"); // match the behavior of syslog-redirector
```

Comment:
We should [mention here](https://github.com/spotify/helios/blob/9b45e0ded17147d9f6081874c80b5bf94a1dd4da/README.md#L227) that syslog-redirector is deprecated as of Docker version XX. Also mention the right way to log to syslog.